### PR TITLE
Add DEV Podcast free stickers opportunity

### PR DESCRIPTION
**Pull request by alsoknownasdrew**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Added free DEV.to stickers pack opportunity described [here](https://dev.to/devteam/introducing-devnews-a-new-podcast-from-dev-3b7m) and [here](https://airtable.com/shr8oKAIMZgdYnBxx)

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -664,5 +664,14 @@
     "image": "http://devblogs.microsoft.com/xamarin/wp-content/uploads/sites/44/2019/10/medium_25686-c9dd82896c9e66bb82600f256444abe6.png",
     "dateAdded": "2019-10-08T21:00:00.000Z",
     "tags": ["stickers", "hacktoberfest", "clothing", "expired"]
+  },
+  {
+    "name": "DEV Podcast",
+    "difficulty": "easy",
+    "description": "Leave a review about DevNews OR DevDiscuss podcasts on Apple Podcasts and get a free pack of DEV stickers ",
```
Comment (aslafy-z) [inline data.json:671]:
```suggestion
    "description": "Leave a review about DevNews OR DevDiscuss podcasts on Apple Podcasts and get a free pack of DEV stickers.",
```

File: data.json [hunk 2]
```diff
@@ -664,5 +664,14 @@
     "image": "http://devblogs.microsoft.com/xamarin/wp-content/uploads/sites/44/2019/10/medium_25686-c9dd82896c9e66bb82600f256444abe6.png",
     "dateAdded": "2019-10-08T21:00:00.000Z",
     "tags": ["stickers", "hacktoberfest", "clothing", "expired"]
+  },
+  {
```
Comment (aslafy-z) [inline data.json:668]: Can you reorder alphabetically please? Thank you

Comment (alsoknownasdrew) [reply data.json]: Yes, sure 👍

**Review by aslafy-z — APPROVED** [review]
LGTM

**Review by vikaspotluri123 — APPROVED** [review]

**Pull request merged**